### PR TITLE
Adds library boilerplate

### PR DESCRIPTION
**Pull request by albertein**
- [x] @pablasso 
- [ ] @rpfernando

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+3. [Usage](#usage)
+4. [Hacking](#hacking)
+    1. [Setup](#setup)
+    2. [Testing](#testing)
+    3. [Coding conventions](#coding-conventions)
+
+
+## About
+
+Relayer is a library to emit kafka message and group logs to relay them to kafka for log aggregation.
```
Comment (pablasso) [inline README.md]: messages

File: README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+## Install
+
+Just do:
```
Comment (pablasso) [inline README.md]: Since it's public we should publish to PyPi once it's feature complete.

(unnecessary "just do")

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        version_regex = r'^__version__\s*=\s*[\'"](.+)[\'"]'
+        return re.search(version_regex, f.read(), re.MULTILINE).group(1)
+
+setup(
+    name='relayer',
+    version=get_version(),
+    url='https://github.com/wizeline/relayer',
+    author='Wizeline',
+    author_email='engineering@wizeline.com',
+    description='Relayer is a library to emit kafka message and group logs to relay them to kafka for log aggregation.',
```
Comment (pablasso) [inline setup.py]: messages

**Comment by pablasso** [comment]
:shipit:

File: README.md (hunk 2 above)
Comment (albertein) [inline README.md]: @pablasso will do as soon as have testing just as `sqlalchemy-pagination` is published

**Comment by albertein** [comment]
Ignoring build failure since this commit doesn't include the lib code yet

**Pull request merged**